apps/index.html

```diff
@@ -28,7 +28,10 @@
         <a href="https://github.com/unhosted/website"><strong>+ add yours</strong></a></p>
       </p>
       <ul class="icons">
-       <li> <a href="http://litewrite.net"><img src="/apps/litewrite.png" /></a>
+       <li> <a href="https://github.com/duniter/cesium"><img src="/apps/cesium.png" /></a>
```

Comment:
Can you link to a URL on which this app is hosted? You can for instance deploy it using 5apps.com or host it on gh-pages, and then link to that URL instead of to the URL of the source code repo.